### DISCUSSION TITLE
Manually trigger a scheduled task

### DISCUSSION THREAD

**Discussion by dwarsbit** [coollabsio/coolify#2621] [Feature Requests]
It would be nice to have the ability to manually trigger scheduled tasks. This can be useful for setting up new tasks and checking if they work; or for retrying failed tasks. I know there is the `command` feature, so it's just a quality of life feature that might be easy to implement.

**Comment by douglasg14b** [comment]
This would be beyond nice.

**Comment by peaklabs-dev** [comment]
This has been added in a recent release:
<img width="901" alt="image" src="https://github.com/user-attachments/assets/60a0d727-ef1b-434f-8de1-6a792967cfb8" />
